meta-xt-prod-cockpit-rcar-driver-domain/recipes-extended/cluster-view/files/ActivatedEvent.cpp

```diff
@@ -5,21 +5,27 @@
 #include <sys/ioctl.h>
 #include <fcntl.h>
 #include <unistd.h>
+#ifndef UNIT_TEST
 #include <linux/r_taurus_cluster_protocol.h>
+#endif
 #include "ActivatedEvent.h"
 
 ActivatedEvent::ActivatedEvent()
 {
+#ifndef UNIT_TEST
```

Comment:
It is better to send unit test data to the cluster and see the result on it

Reply:
it has to be something like : test-mode with the predefined route - will be done